### DISCUSSION TITLE
Studio One Users

### DISCUSSION THREAD

**Discussion by RandyNorton** [apohl79/audiogridder#638] [Help]
Hi, first Congrats for this system, absolutely GREAT. 
To help for the beta 1.2 i am a studioOne user 5.3 Sphera, the 1.1 works fine, stable in plugin mode and in instrument mode, it doesn't the Midi plugin. 1.2 work great only the audio plugin but does't instrument, and midi. (PC Win 10) Instrument vst3 not open channel output and don't create channel. i see plugin work but i can't connect to the mixer, no channels available. Vst2 same issues but there is visibility of multichannel in the flag of plugin but can't check flag to create channel on mixer. In Mac it work AU, create channel in mixer but all system go freeze. Audio plugin insert work fine. Great the sidechain function. Hope to can help developers with my post. If Want help like beta tester i have lot Computers in my studio and i offer my support. Hello to all. Thanks for your time. ( Sorry for my english )

**Comment by apohl79** [comment]
Please try beta 4. Instrument support should work better.

**Reply by RandyNorton** [reply]
Yes, did yesterday, great!! Thanks
